src/App.js

```diff
@@ -1,12 +1,59 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import './App.css';
+import Card from "./components/Card";
+import axios from "axios";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [error, setError] = useState("");
+    const [data, setData] = useState([]);
```

Comment:
Zodra de setter aangeroepen wordt is `data` niet meer van het type array. Ik zou daarom de default waarde dan ook `undefined` maken
```suggestion
    const [data, setData] = useState();
```